src/main/java/org/jenkinsci/plugins/ansible/AnsiblePlaybookBuilder.java

```diff
@@ -85,9 +89,9 @@
 
     @Deprecated
     public AnsiblePlaybookBuilder(String ansibleName, String playbook, Inventory inventory, String limit, String tags,
-                                  String skippedTags, String startAtTask, String credentialsId, boolean sudo,
-                                  String sudoUser, int forks, boolean unbufferedOutput, boolean colorizedOutput,
-                                  boolean hostKeyChecking, String additionalParameters)
+                                  String skippedTags, String startAtTask, String credentialsId, boolean become,
+                                  String becomeUser, boolean sudo, String sudoUser, int forks, boolean unbufferedOutput,
```

Comment:
Are also required? Maybe a migration test would be great (but not required as we also have another constructor with the databoundconstructor annotation)